word-api/app.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const config = require('./config');
+
+const express = require('express');
+const helmet = require('helmet');
+
+const app = express();
+
+// Set headers - see https://helmetjs.github.io/
+app.use(helmet(config.helmet));
+
+// Am I alive?
+app.get('/ping', (request, response) => response.send('PONG'));
+
+// TODO: require routes dir
+app.get('/', (request, response) => response.send('Hello World!'));
+
+// Start it up
+module.exports = app.listen(config.application.port, () => {
+	console.log('word-api - listening on port 5959');
```

Comment:
Use the configured port value rather than hard-coding it.

Reply:
Fixed